.github/workflows/update-bootstrap-deploy-id.yml

```diff
@@ -16,18 +16,17 @@ jobs:
 
       - name: Replace Deploy ID
         run:
-          find ./ -type f -exec sed -i 's/https:\/\/.*--edge-bootstrap\.netlify\.app/https://{{
-          $inputs.bootstrap_deploy_id }}--edge-bootstrap.netlify.app/g' {} \;
+          find ./ -type f -exec sed -i 's@https:\/\/.*--edge-bootstrap\.netlify\.app@https://${{
```

Comment:
How do you feel about replacing this only on specific files? I wonder if this will become annoying if we have a similar URL in a README file or even in another source file where we don't want to do the replacement.

Reply:
Thought about this as well. My reasoning was that it's more likely to have these URLs move to different files, than it is that we have a deploy-specific URL in the README.